internal/v5/auth.go

```diff
@@ -422,9 +422,8 @@ func (h *ReqHandler) checkRequest(p authorizeParams) (authorization, error) {
 
 // entityACLs calculates the ACLs for the specified entity. If the entity
 // has been published to the stable channel then the StableChannel ACLs will be
-// used; if the entity has been published to development, but not stable
-// then the DevelopmentChannel ACLs will be used; otherwise
-// the unpublished ACLs are used.
+// used; if the entity has been published to edge, but not stable then the
```

Comment:
ISTM that this comment isn't strictly accurate any more. If a channel is specified in the request, that channel will be used for ACLs, I think.